packages/vizbuilder/src/components/ChartCard.jsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import {Button} from "@blueprintjs/core";
+
+import {charts} from "../helpers/chartconfig";
+
+class ChartCard extends React.Component {
+  render() {
+    const {
+      active,
+      config,
+      dataset,
+      onSelect,
+      query,
+      type,
+      yearSelector
+    } = this.props;
+    const ChartComponent = charts[type];
+
+    const title = `${type} of ${query.measure.name || ""} by ${query.drilldown
```

Comment:
move this title logic into the chartconfig logic

Reply:
this title is currently not rendered in the chart, but in the component, along with the year selector
should I move this title to the chart itself? otherwise it doesn't make sense to move this logic

Reply:
yes, moving the titles into the chart itself will also help with accessibility, as this will become a hidden `<title>` element inside of the SVG